lua/zipkin.lua

```diff
@@ -90,6 +90,11 @@ function zipkin.emit_syslog(headers, start_time, end_time, response)
             ngx.var.request_uri,
             ngx.var.server_protocol
         )
+        -- For DELETEs which might not have any response,assume spectre_cache_status is MISS 
+        local spectre_cache_status = "MISS"
```

Comment:
Should we lowercase this? Just as the other cache miss values are set to "miss" and may make searching easier, unless this intentionally different / uppercase?

Reply:
sgtm!